bqskit/exec/runners/quest.py

```diff
@@ -278,3 +295,81 @@ def annealing_objective(x: npt.NDArray[np.float64], *args: Any) -> float:
     b_dv = (1 - w) * np.percentile(distances, p)
 
     return n_cx + b_dv
+
+
+def gen_approximate_circuits(
+    circuit: Circuit,
+    block_size: int = 3,
+    compiler: Compiler | None = None,
+    weit: float = 0.5,
+    pert: int = 100,
+    approx_threshold: float | None = None,
+    sample_size: int = 16,
+    instantiate_options: dict[str, Any] = {}
+) -> list[Circuit]:
+    """
+    Use the QUEST algorithm to generate approximate circuits.
+
+    Args:
+        block_size (int): The maximum number of qudits in each
+            block. Defaults to 3.
+
+        compiler (Compiler | None): The compiler object used
+            to partitioning and synthesize the circuit. If left
+            as None, then a standard compiler will be created.
+
+        weit (float): The weight set on number of CNOTs versus
```

Comment:
I would write out the full name `weight` here, its not much longer, and I've never seen `weit` before. Same with `pert` below.

Reply:
I agree with you, I was sticking with the vocab used in the prototype QuEST algorithm. I think we should keep the vocab matching, even if we aren't fans of it.